src/Symfony/Bundle/FrameworkBundle/Test/BrowserKitAssertionsTrait.php

```diff
@@ -38,6 +38,11 @@ public static function assertResponseStatusCodeSame(int $expectedCode, string $m
         self::assertThatForResponse(new ResponseConstraint\ResponseStatusCodeSame($expectedCode), $message);
     }
 
+    public static function assertResponseFormatSame(?string $expectedFormat, string $message = ''): void
```

Comment:
```suggestion
    public static function assertResponseFormatSame(string $expectedFormat, string $message = ''): void
```

?

Reply:
I think we should keep it as is to be able to pass directly the return of `Request::getRequestFormat()`. This also allows testing if the format is `null` (even if it's probably a niche use case).

Reply:
Could you add a testcase for null then? Thanks